### PR TITLE
Add Usage Section in README.md

### LINKED ISSUE #24: [Feature] Updating README.md with the new section for flag options
**Please describe the feature you want this project to offer**
After adding the flag parsing functionality to the program, the README.md file does not reflect the usage of it, as was done in PR #17  

**Describe the solution you'd like**
We can have a proper section for the usage of this program, such as an example block in the README.md file

### PR DESCRIPTION
**Pull request by sjchoure**
Fixes #24 

- [x] Added Usage Section for providing information on Flag usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,72 @@ Run the following commands from the root of the project directory:
...
+                IPv6 -> 
+                IPv4 -> 
+        [WiFi] 
+                IPv6 -> 
+                IPv4 -> 
+        [Loopback Pseudo-Interface 1]
+                IPv6 -> ::1/128
+                IPv4 -> 127.0.0.1/8
+Public IPv4 Address: 
+```
```
Comment (DarthBenro008) [inline README.md]: Maybe we can a small screenshot rather than a console output, we have coloured outputs

**Review by DarthBenro008 — COMMENTED** [review]
I think this README update can be done after the basic functions have been fully implemented.

**Comment by sjchoure** [comment]
> I think this README update can be done after the basic functions have been fully implemented.

Okay, saving the PR as Draft

**Comment by DarthBenro008** [comment]
@saviourcode could you rebase the readme and make the changes mentioned before?

**Comment by sjchoure** [comment]
@DarthBenro008 Yes sure.

File: README.md (hunk 1 above)
Comment (sjchoure) [reply README.md]: Will it be okay to create a Screenshot folder to store the images/screenshots required? Like Assets?

**Comment by sjchoure** [comment]
@DarthBenro008 @aryan9600 I have made the desired changes to the README.md file, link to preview: [here](https://github.com/saviourcode/termiboard/blob/readme-flag/README.md)

File: README.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ Kindly consider leaving a :star: if you like the repository and our organisation
 
 * Run termiboard
 
-```
+```console
 go run *.go
```
Comment (DarthBenro008) [inline README.md]: This should be changed to ```go run .``` instead as we have test files now

File: README.md [hunk 3]
```diff
@@ -45,10 +45,40 @@ Run the following commands from the root of the project directory:
 
 Open your forked repo in your browser and then raise a PR to the `master` branch of this repository!
 
+## Usage
+Once the executable file is created, the program can be executed as:
 
+### Without Flag Options
+This will by default show all the parameters of the System. It will provide all the parameters of CPU, Memory and Network Status.
+```console
```
Comment (DarthBenro008) [inline README.md]: please re-evaluate the punctuations.

File: README.md [hunk 4]
```diff
@@ -45,10 +45,40 @@ Run the following commands from the root of the project directory:
...
+| Flag      | Function                                        |
+|-----------|-------------------------------------------------|
+| all       | Show all stats                                  |
+| cpu-info  | Show CPU information                            |
+| cpu-usage | Show CPU usage                                  |
+| disk      | Show disk usage                                 |
+| local-ip  | Show local IP address                           |
+| public-ip | Show public IP address                          |
+| ram       | Show RAM usage                                  |
+| top5-ram  | Show top 5 process that consume the most memory |
```
Comment (DarthBenro008) [inline README.md]: i feel you can append the prefix -- to all the rows in the flag coulmn

**Review by DarthBenro008 — CHANGES_REQUESTED** [review]
Please refactor all the assets inside a folder name "assets/" and make the requested changes, everything else LGTM! Nice Work :)

File: README.md (hunk 2 above)
Comment (sjchoure) [reply README.md]: Okay sure

File: README.md (hunk 3 above)
Comment (sjchoure) [reply README.md]: Okay

File: README.md (hunk 4 above)
Comment (sjchoure) [reply README.md]: Yes, okay

**Comment by sjchoure** [comment]
@DarthBenro008 I am adding badges

**Comment by sjchoure** [comment]
Badges
@DarthBenro008 Are they covering our project properly?

**Comment by DarthBenro008** [comment]
Open Source , license , go version, build status from termiboard-build CI, open issues, PRs welcome. These badges would be enough and please maintain consistent badge styles , the PRs welcome badge has a different styling than others

**Comment by sjchoure** [comment]
badgesnew

**Review by DarthBenro008 — APPROVED** [review]
LGTM! awesome work @saviourcode !

**Pull request merged**